Templates/TypeScript/EnumType.stencil

```diff
@@ -5,9 +5,4 @@ export enum {{ enumType.name }} {
   {% with enumValue.attributes as attributes %}{% include "Helpers/Attributes.stencil" %}{% endwith %}
   {{ enumValue.value|pascalCase }} = "{{ enumValue.value }}",
   {% endfor %}
-
-  /**
-   * Unknown Syrup enum.
-   */
-  UnknownSyrupEnum = "UNKNOWN_SYRUP_ENUM"
```

Comment:
Without the unknown type, what happens if we get an enum value from the backend that's not a valid enum case? I'm guessing before we were never actually getting `UnknownSyrupEnum` values since we would assign the raw string in our response to the enum type?